machine_configuration/mco-coreos-layering.adoc

```diff
@@ -76,8 +76,8 @@ content: |-
 # Using a 4.12.0 image
```

Comment:
I do not think 4.12 will support this, as this is for 4.17+, maybe 4.17.0 is better? CC @jlebon

Reply:
Yes, you're right. We should update this to be 4.18/4.17.